controllers/auth.js

```diff
@@ -55,21 +55,32 @@ const User = require('../models/User')
   }
   
   exports.postSignup = (req, res, next) => {
-    const validationErrors = []
-    if (!validator.isEmail(req.body.email)) validationErrors.push({ msg: 'Please enter a valid email address.' })
-    if (!validator.isLength(req.body.password, { min: 8 })) validationErrors.push({ msg: 'Password must be at least 8 characters long' })
-    if (req.body.password !== req.body.confirmPassword) validationErrors.push({ msg: 'Passwords do not match' })
-  
+    const validationErrors = [];
+    if (!validator.isEmail(req.body.email))
+        validationErrors.push({ msg: "Please enter a valid email address." });
+    if (!validator.isLength(req.body.password, { min: 8 }))
+        validationErrors.push({
+            msg: "Password must be at least 8 characters long",
+        });
+    if (req.body.password !== req.body.confirmPassword)
+        validationErrors.push({ msg: "Passwords do not match" });  
+    if (req.body.firstName !== req.body.firstName)
```

Comment:
lines 67 -70:

If the firstName and lastName fields are left blank they will still match (undefined === undefined).

Also I would suggest minor changes to the text in the messages: `Name` to `first name` and `last Name` to `last name`
E.g.,
```
    if (!req.body.firstName)
      validationErrors.push({ msg: 'Please enter your Name.' }) 
    if (!req.body.lastName)
      validationErrors.push({ msg: 'Please enter your last Name.' }) 
```

Reply:
Updated the name conditional here ([5d08e41](https://github.com/CadmiumRed40/FindAFlick/commit/5d08e41a8506a059ba8994b6e63f8a9bc2e43b08))

Also updated the text in `msg ` for both.

Reply:
There is a bug in commit [5d08e41](https://github.com/CadmiumRed40/FindAFlick/commit/5d08e41a8506a059ba8994b6e63f8a9bc2e43b08).

`req.body.firstName` doesn't have an attribute `min.length`. Instead, you can write it like the snippet below. I also updated the msg to be more accurate since it will kick out names of length 1.

```
    if (!validator.isLength(req.body.firstName, { min: 2 }))
      validationErrors.push({ msg: 'First name must be at least 2 characters.' })
    if (!validator.isLength(req.body.lastName, { min: 2 }))
      validationErrors.push({ msg: 'Last name must be at least 2 characters.' })
```